src/OpenTelemetry.Exporter.Jaeger/JaegerExporter.cs

```diff
@@ -58,7 +57,9 @@ internal JaegerExporter(JaegerExporterOptions options, TTransport clientTranspor
             this.memoryTransport = new InMemoryTransport(16000);
             this.memoryProtocol = this.protocolFactory.GetProtocol(this.memoryTransport);
 
-            this.Process = new Process(DefaultServiceName, options.ProcessTags);
+            string serviceName = (string)this.ParentProvider.GetDefaultResource().Attributes.Where(
```

Comment:
add a note in Jaegers changelog as well. (This is not public API change, but behavior change)